vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -74,6 +82,7 @@ func init() {
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
 	StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.Beta},
+	ValidateProxyRedirects:  {Default: false, PreRelease: utilfeature.Alpha},
```

Comment:
it's `off` by default.

Reply:
it looks like it could be manually enabled with the kube-apiserver flag `--feature-gates=ValidateProxyRedirects=true`

Reply:
would set default to true and add a comment

Reply:
done